packages/cli/api-importers/openapi/openapi-ir-to-fern-tests/src/__test__/__snapshots__/openapi/suger.json

```diff
@@ -2372,12 +2372,12 @@ in the google.rpc.Status.details field, or localized by the client.",
               "displayUnit": "optional<string>",
               "displayUnitDescription": "optional<string>",
               "id": {
-                "docs": "The usage metering dimension key, in format of "{plan_key}_{usage_dimension_key}"",
+                "docs": "The usage metering dimension key, in format of "{plan*key}*{usage_dimension_key}"",
                 "type": "optional<string>",
               },
               "metricKind": "optional<string>",
               "name": {
-                "docs": "Name: The resource name of the metric descriptor, in format of "{service_name}/{plan_key}_{usage_dimension_key}"",
+                "docs": "Name: The resource name of the metric descriptor, in format of "{service*name}/{plan_key}*{usage_dimension_key}"",
```

Comment:
Running this on more test definitions illustrates that we might not actually want to run `prettier` in general. 

It seems like the main thing we want to do is clean up leading/trailing whitespace, but even that might best be shipped in a separate formatter tool that users can independently control. Otherwise we might end up with surprising behavior like this.

Reply:
I removed `prettier` altogether in [this commit](https://github.com/fern-api/fern/pull/5363/commits/7f04efccd2998cf1a438bcb482c12b98b8463132) to explore the results a bit more.

Reply:
Discussed offline - we'll remove `prettier` altogether for these reasons.